src/drivers/esp8266_drv.c

```diff
@@ -199,6 +199,15 @@ static void cmd_started()
         esp8266_state.cmd.in_progress = true;
 }
 
+static void check_excessive_failures(void)
+{
+        if (esp8266_state.cmd.failures >= CMD_MAX_FAILURES) {
```

Comment:
Personal preference here... I always use rapid returns to avoid excess indentation. Also better to use Yoda conditionals to prevent accidental assignments. So instead I would do this:
```
if (CMD_MAX_FAILURES > esp8266_state.cmd.failures ) return;
```

Reply:
thanks, applying this change.